dts/bindings/dma/st,stm32-dma-v3.yaml

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2021 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  STM32 DMA controller (V3)
+
+  It is present on stm32 devices stm32U5
+  This DMA controller includes several channels with different requests.
+  DMA clients connected to the STM32 DMA controller must use the format
+  described in the dma.txt file, using a four-cell specifier for each
+  capable of supporting 15 dependent DMA channels.
+  DMA clients connected to the STM32 DMA controller must use the format
+  described in the dma.txt file, using a 3-cell specifier for each
+  channel: a phandle to the DMA controller plus the following four integer cells:
+    1. channel: the dma stream from 1 to <dma-requests>
+    2. slot: DMA periph request ID, which is written in the DMAREQ_ID of the DMAMUX_CxCR
+    this value is 0 for Memory-to-memory transfers
+    or a value between <1> .. <dma-generators> (not supported yet)
+    or a value beweeen <dma-generators>+1  ..  <dma-generators>+<dma-requests>
+    3. channel-config: A 32bit mask specifying the DMA channel configuration
+    which is device dependent:
+        -bit 6-7 : Direction  (see dma.h)
+               0x0: MEM to MEM
+               0x1: MEM to PERIPH
+               0x2: PERIPH to MEM
+               0x3: reserved for PERIPH to PERIPH
+        -bit 9 : Peripheral Increment Address
+               0x0: no address increment between transfers
+               0x1: increment address between transfers
+        -bit 10 : Memory Increment Address
+               0x0: no address increment between transfers
+               0x1: increment address between transfers
+        -bit 11-12 : Peripheral data size
+               0x0: Byte (8 bits)
+               0x1: Half-word (16 bits)
+               0x2: Word (32 bits)
+               0x3: reserved
+        -bit 13-14 : Memory data size
+               0x0: Byte (8 bits)
+               0x1: Half-word (16 bits)
+               0x2: Word (32 bits)
+               0x3: reserved
+        -bit 15: Reserved
+        -bit 16-17 : Priority level
+               0x0: low
+               0x1: medium
+               0x2: high
+               0x3: very high
+  Example of dma node for stm32wb55x
+     dma2: dma-controller@40020400 {
+         compatible = "st,stm32-dma-v2";
+         ...
+         dma-requests = <7>;
+         status = "disabled";
+         label = "DMA_2";
+        };
+
+  For the client part, example for stm32l476 on DMA1 instance
+    Tx using channel 3 with request 1
+    Rx using channel 2 with request 1
+    spi1 {
+     compatible = "st,stm32-spi";
+     dmas = <&dma1 3 1 0x20440>,
+            <&dma1 2 1 0x20480>;
+     dma-names = "tx", "rx";
+    };
```

Comment:
Please update example to fit this v3 binding

Reply:
done